go/vt/schemadiff/table.go

```diff
@@ -482,6 +482,7 @@ func (c *CreateTableEntity) normalizeIndexOptions() {
 		idx.Info.Type = strings.ToLower(idx.Info.Type)
 		for _, opt := range idx.Options {
 			opt.Name = strings.ToLower(opt.Name)
+			opt.String = strings.ToLower(opt.String)
```

Comment:
Hmm, would this always lowercase things if you have for example a comment set? It seems like we should not normalize that then. I wonder if we should make this specific only to `WITH PARSER`?

Reply:
Per our parser:

https://github.com/vitessio/vitess/blob/1ace3b47046a40e4c43b1ee0d2088db01f35ef0d/go/vt/sqlparser/sql.y#L2299-L2332

It looks like the `String` value is only ever used (at this time?) in a `WITH PARSER` clause. No other index clause/option utilizes it.

Reply:
hmmm also here:

https://github.com/vitessio/vitess/blob/1ace3b47046a40e4c43b1ee0d2088db01f35ef0d/go/vt/sqlparser/sql.y#L7240-L7244

`USING ...`, so this affects e.g. `USING HASH` -> `USING hash`

Reply:
I don't see any issues with lower-casing `USING` or `WITH PARSER` values